DEVELOPMENT.md

```diff
@@ -1,5 +1,15 @@
 # Development
 
+## Requirements
+
+* Vagrant - 2.2+
+* Ansible - 2.12+
```

Comment:
```suggestion
* Ansible - 2.14+
```

2.14 is what is currently in CentOS Stream 9 and what we use in CI -- I'd not document anything older.